docs/blog/newsletter-november-2024.rst

```diff
@@ -0,0 +1,73 @@
+
+Greetings, documentarians. In these bustling days of news and planning, I hope you are able to find moments of calm for yourselves to appreciate what is going well.
+
+If you have enough calm and want some inspiration, good news: The Write the Docs Australia conference is coming soon! Tickets close on Saturday 16 November, so `get your tickets <https://www.writethedocs.org/conf/australia/2024/tickets/>`__ while you still can.
+
+The `WTD Salary Survey for 2024 <https://salary-survey.writethedocs.org/>`__ is now open! New this year: you can fill the survey out as both an employee and contractor, if you've worked as both during the past year; answers can be edited before the final submission; and for privacy, salary/rate fields are masked by default. Head on over and share your insights – each contribution helps build a clearer picture of what fair, equitable, and transparent compensation looks like in our industry. Full-time or part-time, permanent or contract, working or currently between jobs – we want your input!
+
+And in the newsletter this month, we have insights into whether to use emoji in docs, whether you need to know a tool before applying to work with it, and dealing with assets with docs-as-code. Enjoy!
+
+----------------------------------------------
+Emoji in documentation: :yes-please: or :yuk:?
+----------------------------------------------
+
+Emoji seem to find their way into documentation more and more often these days... but is that a good or a bad idea? Well, like everything else, it depends! A sprinkling of emoji can work well in some cases and cause more problems than they're worth in others.
+
+Emoji can help achieve an inviting and approachable tone for certain types of documentation, such as quick start guides aimed at developers. Because emoji can make content feel less dense and more engaging, they may be especially appropriate for contexts such as presales documentation, where a whiff of marketing-speak is acceptable. Emoji may also be used to differentiate a newcomer's product from existing enterprise offerings. And emoji in tables can make the information easier for readers to scan, although icons used as inline images can achieve the same thing.
+
+On the other hand, emoji can cause accessibility issues because they aren't necessarily interpretable by assistive technology (read `Do emoji and accessibility work together? <https://www.tiny.cloud/blog/emoji-and-accessibility/#accessible-emoji>`_ by Di Mace for details). The meanings of emoji may be interpreted differently across cultures, so you risk sending a different message than you intended when you use emoji. Emoji can be distracting or even inconsiderate to a confused reader. Documentarians may also spend a lot of time trying to use emoji to affect a chatty tone instead of writing clearly and end up with content that doesn't actually explain anything. It's best to carefully consider your audience and approach emoji use with caution.
```

Comment:
```suggestion
On the other hand, emoji can cause accessibility issues because they aren't necessarily interpretable by assistive technology. (For details, read `Do emoji and accessibility work together? <https://www.tiny.cloud/blog/emoji-and-accessibility/#accessible-emoji>`_ by Di Mace.) The meanings of emoji may be interpreted differently across cultures, so you risk sending a different message than you intended when you use emoji. Emoji can be distracting or even inconsiderate to a confused reader. Documentarians may also spend a lot of time trying to use emoji to affect a chatty tone instead of writing clearly and end up with content that doesn't actually explain anything. It's best to carefully consider your audience and approach emoji use with caution.
```